handler/perm_handlers.go

```diff
@@ -0,0 +1,80 @@
+package handler
+
+import (
+	"github.com/gin-gonic/gin"
+	"net/http"
+	"stlab.itechart-group.com/go/food_delivery/authorization_service/model"
+	"strconv"
+)
+
+// @Summary createPerm
+// @Tags permission
+// @Description create new permission
+// @Accept  json
+// @Produce  json
+// @Param input body model.CreatePerm true "Perm"
+// @Success 201 {string} string
+// @Failure 400 {string} string
+// @Failure 500 {string} string
```

Comment:
Does this doc say that you return strings on error? As I understand you return object with message field